tools/packaging/kernel/configs/fragments/common/dax.conf

```diff
@@ -12,7 +12,6 @@ CONFIG_SPARSEMEM_VMEMMAP=y
 # Without these the pmem_should_map_pages() call in the kernel fails with new
 # Related to the ARCH_HAS_HMM set in the arch files.
 CONFIG_ZONE_DEVICE=y
-CONFIG_DEV_PAGEMAP_OPS=y
```

Comment:
Why are you removing this one?

Reply:
CONFIG_DEV_PAGEMAP_OPS is no longer configurable: https://www.kernelconfig.io/config_dev_pagemap_ops

Will add to commit message.